sbom_generation.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Do update the copyright year to 2023-2024

Reply:
Updated the commit